pkgs/os-specific/linux/ffado/default.nix

```diff
@@ -62,25 +73,15 @@ stdenv.mkDerivation rec {
 
   # TODO fix ffado-diag, it doesn't seem to use PYPKGDIR
   buildPhase = ''
-    export PYDIR=$out/lib/${python.libPrefix}/site-packages
-
-    scons PYPKGDIR=$PYDIR DEBUG=False \
-      ENABLE_ALL=True \
-      SERIALIZE_USE_EXPAT=True \
+    scons ${SCONS_OPTIONS}
   '';
 
-  installPhase = if libOnly then ''
-    scons PREFIX=$TMPDIR UDEVDIR=$TMPDIR \
-      LIBDIR=$out/lib INCLUDEDIR=$out/include install
-  '' else ''
-    scons PREFIX=$out PYPKGDIR=$PYDIR UDEVDIR=$out/lib/udev/rules.d install
-  '' + stdenv.lib.optionalString (optPyqt4 != null && optPythonDBus != null) ''
+  installPhase = ''
+    scons ${SCONS_OPTIONS} ${if mixerOnly then "${BINDIR} ${PYDIR}" else "install"}
```

Comment:
You don't have to do this here, but eventually I want to get scons to have setup hook that defines these phases (like is done with cmake, ninja, etc...).

Reply:
IDK other legal way to bypass library build, when we build only mixer. `scons $out/lib/python $out/bin` force installing only needed stuff.

Reply:
@matthewbauer what can I do, to get this one PR merged? Is a deepest rebuild trigger in my WIP tree at the moment.  (probably we can even drop mixer temporary, and make separate package for it, if anyone need it really)